content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -62,7 +62,7 @@ _Labels_ are key/value pairs. Valid label keys have two segments: an optional
 prefix and name, separated by a slash (`/`). The name segment is required and
 must be 63 characters or less, beginning and ending with an alphanumeric
 character (`[a-z0-9A-Z]`) with dashes (`-`), underscores (`_`), dots (`.`),
-and alphanumerics between. The prefix is optional. If specified, the prefix
+and alphanumerics allowed in between. The prefix is optional. If specified, the prefix
```

Comment:
IMO
I don't feel this updated text adding any value, current existing context looks fine.